mindee/client.py

```diff
@@ -194,6 +195,98 @@ def parse_queued(
 
         return self._get_queued_document(product_class, endpoint, queue_id)
 
+    def _validate_async_params(
+        self, initial_delay_sec: float, delay_sec: float
+    ) -> None:
+        if delay_sec < 2:
+            raise TypeError("Cannot set auto-parsing delay to less than 2 seconds.")
+        if initial_delay_sec < 4:
+            raise TypeError("Cannot set initial parsing delay to less than 4 seconds.")
+
+    def enqueue_and_parse(
+        self,
+        product_class,
+        input_source: Union[LocalInputSource, UrlInputSource],
+        include_words: bool = False,
+        close_file: bool = True,
+        page_options: Optional[PageOptions] = None,
+        cropper: bool = False,
+        endpoint: Optional[Endpoint] = None,
```

Comment:
when do we need to set the endpoint (honest question)?

Reply:
Does this imply that other questions were dishonest? :thinking: 
Jokes aside, we don't need it atm, but I thought I might as well make it compatible for potential future custom async builds? It doesn't hurt, and that way it looks like the other functions.